### PR TITLE
[DOCFIX] Introduce alluxio-dev in Docker doc

### PR DESCRIPTION
**Pull request by ssz1997**
Add a short paragraph to introduce alluxio/alluxio-dev. What it is for and what's the difference between it and alluxio/alluxio.

### CODE REVIEW

File: docs/en/deploy/Running-Alluxio-On-Docker.md [hunk 1]
```diff
@@ -284,6 +284,13 @@ Congratulations, you've deployed a basic Dockerized Alluxio cluster! Read on to
 
 ## Advanced Setup
 
+### Launch Alluxio with the development image
+Starting from v2.6.2, we have another docker image, alluxio-dev, available in Dockerhub for development usage. Unlike the default alluxio/alluxio image that
```
Comment (LuQQiu) [inline docs/en/deploy/Running-Alluxio-On-Docker.md:288]: Extra line between title and paragraph

Comment (LuQQiu) [reply docs/en/deploy/Running-Alluxio-On-Docker.md:288]: Starting from v2.6.2, a new docker image, alluxio-dev, is available  ....
try to reduce we/I in the doc

File: docs/en/deploy/Running-Alluxio-On-Docker.md [hunk 2]
```diff
@@ -284,6 +284,13 @@ Congratulations, you've deployed a basic Dockerized Alluxio cluster! Read on to
 
 ## Advanced Setup
 
+### Launch Alluxio with the development image
+Starting from v2.6.2, we have another docker image, alluxio-dev, available in Dockerhub for development usage. Unlike the default alluxio/alluxio image that 
+only contains packages needed for Alluxio service to run, this alluxio-dev image installs more development tools, including gcc, make, async-profiler, etc., 
+making it easier for users to deploy more services in the container along with Alluxio. 
+
+To use the development image, simply replace alluxio/alluxio with alluxio/alluxio-dev in the container launching process.
```
Comment (LuQQiu) [inline docs/en/deploy/Running-Alluxio-On-Docker.md:292]: treat alluxio/alluxio alluxio/alluxio-dev as code
same for all other places mention the docker image name

**Review by LuQQiu — COMMENTED** [review]
Thanks for adding the doc

File: docs/en/deploy/Running-Alluxio-On-Docker.md [hunk 3]
```diff
@@ -284,6 +284,14 @@ Congratulations, you've deployed a basic Dockerized Alluxio cluster! Read on to
 
 ## Advanced Setup
 
+
+### Launch Alluxio with the development image
+Starting from v2.6.2, a new docker image, `alluxio-dev`, is available in Dockerhub for development usage. Unlike the default `alluxio/alluxio` image that
```
Comment (LuQQiu) [inline docs/en/deploy/Running-Alluxio-On-Docker.md:289]: Add a new line before this line?

**Review by LuQQiu — APPROVED** [review]
LGTM after the line fixed, thanks!

File: docs/en/deploy/Running-Alluxio-On-Docker.md (hunk 3 above)
Comment (ssz1997) [reply docs/en/deploy/Running-Alluxio-On-Docker.md:289]: Ah sorry

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Comment by LuQQiu** [comment]
alluxio-bot, cherry-pick this to release-2.6.2 please

**Pull request merged**